docs/MapColonies/Raster/services/polygon_parts.md

```diff
@@ -0,0 +1,877 @@
+---
+id: raster-polygon-parts-service
+slug: polygon-parts
+title: Polygon Parts 🧩
+description: Search, Filter and Aggregate polygon parts (vectorial) layers
+tags:
+  - wfs
+  - raster
+  - features
+  - API
+  - polygon-parts
+---
+
+# Polygon Parts Service 🧩
+
+The raster polygon parts service uses the [WFS](/docs/ogc/protocols/ogc-wfs) protocol which is a READ-ONLY Web Feature Service.
+it provides facilities for searching and retrieving feature data with the `GetCapabilities`, `DescribeFeatureType` and `GetFeature` operations all other operations are invalid.
```

Comment:
can't you point to the WFS general explanation instead of repeating it?

Reply:
not sure I understand, it already includes referencing to general, I only mentioned again the main API's protocols

![image](https://github.com/MapColonies/developer-portal/assets/72082238/d3a8b4f7-9532-485c-b0f2-241f1a910d16)